pkgs/misc/drivers/xow/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, libusb, git }:
+
+stdenv.mkDerivation rec {
+  pname = "xow";
+  version = "0.2";
+
+  src = fetchurl {
+    url = "https://github.com/medusalix/xow/archive/v${version}.tar.gz";
+    sha256 = "0a7621i4ig6wmd8gyq1lcfxgl4adbq8h2640yvj33cs858d6iz70";
+  };
+
+  makeFlags = "BUILD=RELEASE";
+  enableParallelBuilding = true;
+  buildInputs = [ libusb git ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp xow $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/medusalix/xow;
```

Comment:
Quoting URLs is preferred nowadays, see https://github.com/NixOS/rfcs/pull/45

Reply:
Fixed.